zulip_bots/zulip_bots/bots/followup/followup.py

```diff
@@ -22,15 +23,21 @@ def usage(self):
             called "followup" that your API user can send to.
             '''
 
+    def initialize(self, bot_handler):
+            self.config_info = bot_handler.get_config_info('stream_detail', optional=True)
+            self.default_stream = self.config_info.get("stream", False)
+            self.default_stream = 'followup'
```

Comment:
Hey. It seems like this line would nullify the previous line. Nice call deciding that followup should be the default stream.

You should rewrite this as `self.default_stream = self.config_info.get("stream", "folllowup")` as the second argument to `get` is the default value to return in case we do not have the specified key in config.